migrations.tf

```diff
@@ -0,0 +1,9 @@
+################################################################################
+# Migrations: v2.6.0 -> v3.0.0
+################################################################################
+
+# Node IAM role
```

Comment:
```suggestion
```

Reply:
Good catch, sorry about the wrong copy/paste.